__tests__/Auth0Client/getTokenSilently.test.ts

```diff
@@ -1640,7 +1633,154 @@ describe('Auth0Client', () => {
       await expect(
         auth0.getTokenSilently({ ignoreCache: true })
       ).rejects.toHaveProperty('error', 'login_required');
+
       expect(auth0.logout).toHaveBeenCalledWith({ localOnly: true });
+      windowSpy.mockRestore();
+    });
+
+    it('returns the full token response when returnMode = "verbose"', async () => {
```

Comment:
We will need to change the test descriptions, this is still using `returnMode = 'verbose'`, while at the moment the property is called `verboseReponse`.

Reply:
Fixed the test descriptions 👍🏻